buildscripts/toml-updater-config.gradle

```diff
@@ -26,3 +26,21 @@ versionCatalogUpdate {
         keepUnusedPlugins.set(true)
     }
 }
+
+def isNonStable = { String version ->
+    def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
```

Comment:
Where do these (potentially) show up? Is it reading something from maven? Otherwise, I'd think none of our versions would match since the stable releases are things like 1.2.3.

Reply:
Oh good call—we don't need this line. This was a copy from https://github.com/android/compose-samples/blob/main/JetNews/buildscripts/toml-updater-config.gradle#L30